src/renderer/components/PathInput.tsx

```diff
@@ -30,9 +32,13 @@ export const PathInput = React.forwardRef<HTMLInputElement, PathInputProps>((pro
         <CustomInput inputRef={ref} disabled={true} value={value} placeholder={placeholder} />
         {endAdornment}
       </InputContainer>
-      <Button color="secondary" variant="contained" onClick={onClick} disabled={disabled}>
-        Select
-      </Button>
+      <Tooltip title={tooltipText ?? ""}>
+        <span>
```

Comment:
Why do we need to wrap the `Button` in a span?

Reply:
I think it could be a div or a span, but the reason is because if the Button is disabled the tooltip won't show

Reply:
This is correct. The tooltip will also be disabled if it's immediate child is disabled. Adding an empty wrapper element is the suggested solution.

https://mui.com/components/tooltips/